echopype/convert/set_groups_azfp.py

```diff
@@ -167,6 +167,22 @@ def set_beam(self) -> xr.Dataset:
         ping_time = self.parser_obj.ping_time
         channel_id = self._create_unique_channel_name(unpacked_data)
 
+        # creates unique channel name
+        if unpacked_data["serial_number"].size == 1:
+            freq_as_str = unpacked_data["frequency"].astype(int).astype(str)
+
+            # TODO: replace str(i+1) with Frequency Number from XML
+            channel_id = [
+                str(unpacked_data["serial_number"]) + "-" + freq_as_str[i] + "-" + str(i + 1)
+                for i in range(len(freq_as_str))
+            ]
+        else:
+            channel_id = None
+            raise NotImplementedError(
+                "Creating a channel name for more than"
+                + " one serial number has not been implemented."
+            )
+
```

Comment:
Is this block related to the 0.5x to 0.6x conversion functionality? It doesn't seem to be, as far as I can tell. 

I'm not suggesting there's anything inherently wrong with it. I just want to be clear about its purpose. It seems like it's an improvement unrelated to the core of this PR. Correct?

Reply:
This block and the next one seem to be relics from sometime before? since the content is identical with what's in `_create_unique_channel_name`

Reply:
Yes, this should not be here. Thank you for catching this! It was caused by a merging of branches. I will go ahead and remove it.